shared/tinyusb/tusb_config.h

```diff
@@ -90,7 +90,7 @@
 
 #define USBD_MAX_POWER_MA (250)
 
-#define USBD_DESC_STR_MAX (20)
+#define USBD_DESC_STR_MAX (40)
```

Comment:
In 783ddfc264c8c36253a4279ef501f02f8cac1200 I made this configurable, so you can just increase it for your board in its `mpconfigboard.h` file.